admin/src/locales/en.json

```diff
@@ -126,11 +127,26 @@
       "title": "Delete creation!",
       "yes": "Yes, delete and save creation!"
     },
+    "deleteUser": {
+      "question": "Do you really want to delete {username}?",
+      "title": "Delete user",
+      "yes": "Yes, delete user"
```

Comment:
same

Reply:
For the open contribution table's overlay these more detailed texts for the ok button where introduced to clearly show the admin, what action is behind the buton.

The modals in this PR are meant to adapt to this informative button text approach - for coherence in the admin interface.